packages/odyssey-react/src/components/Infobox/Infobox.module.scss

```diff
@@ -20,21 +20,23 @@
     ". actions";
   grid-template-columns: max-content 1fr;
   width: 100%;
-  max-width: calc(#{$max-line-length} + #{$spacing-m} + #{$spacing-m});
-  padding-block: $spacing-m;
-  padding-inline: $spacing-m;
-  column-gap: $spacing-s;
+  max-width: calc(
+    var(--FontLineLengthMax) + var(--SpacePadding) + var(--SpacePadding)
+  );
```

Comment:
`var(--SpacePadding) * 2` yes?

Reply:
That's better, I think. Tooltip uses a nested `calc()`, and it feels cleaner. I'll update before mergin'.